### PR TITLE
[FIX] Remove empty lines in partner's display address

### PR DESCRIPTION
**Pull request by StefanRijnhart**
as well as removing lines that only contain a comma.

Upstream PR https://github.com/odoo/odoo/pull/9149

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1:

File: openerp/addons/base/tests/test_display_address.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from openerp.tests import common
+
+
+class TestDisplayAddress(common.TransactionCase):
+    def test_display_address(self):
+        """ _display_address() should not include empty lines when some data is missing """
```
Comment (elicoidal) [inline openerp/addons/base/tests/test_display_address.py:6]: s/some data is missing /data are missing

Comment (StefanRijnhart) [inline openerp/addons/base/tests/test_display_address.py:6]: What if I used _data_ as uncountable? http://www.oxforddictionaries.com/definition/learner/data. Would it help if I changed to _some of the data is missing_?

**Comment by yvaucher** [comment]
:+1:

**Comment by yvaucher** [comment]
I consider @elicoidal question answered, plus, it is in tests and it seems correct to me.

**Comment by elicoidal** [comment]
fine by me :)

**Pull request merged**